.github/workflows/check.yml

```diff
@@ -13,34 +13,98 @@ jobs:
+        EMBULK_OUTPUT_MYSQL_TEST_CONFIG: "${{ github.workspace }}/ci/mysql.yml"
+    - uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: mysql5_7
+        path: embulk-output-mysql/build/reports/tests/test
+  mysql8_3:
+    runs-on: ubuntu-latest
+    # push: always run.
+    # pull_request: run only when the PR is submitted from a forked repository, not within this repository.
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      fail-fast: false
+    services:
+      mysql:
+        # Due to MySQL 8.4 disabled mysql_native_password by default,
+        # Connector/J 5.x can't connect to database.
+        # So, Use MySQL 8.3.
+        image: mysql:8.3
+        options: --health-cmd "mysqladmin ping -h localhost" --health-interval 20s --health-timeout 10s --health-retries 10
+        ports:
+        - "3306:3306"
         env:
-          _JAVA_OPTIONS: "-Xmx2048m -Xms512m"
-          EMBULK_OUTPUT_MYSQL_TEST_CONFIG: "${{ github.workspace }}/ci/mysql.yml"
-      - uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: mysql
-          path: embulk-output-mysql/build/reports/tests/test
-  postgresql:
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_USER: ci
+          MYSQL_PASSWORD: password
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 8
+      uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: 'zulu'
+        cache: "gradle"
+    - name: Connect
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "show databases;"
+    - name: show version
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "select version();"
+    - name: Create database
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "create database ci_test;"
+    #
+    # MySQL 8 uses caching_sha2_password mechanism by default.
+    # Connector/J 5.x doesn't support it.
+    #
+    # This part change password mechanism to mysql_native_password.
+    # Remove the following part after update Connector/J
+    #
+    - name: Show password plugins
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "SELECT user, host, plugin FROM mysql.user;"
+    - name: Change password mechanism1 (root@localhost)
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'root';"
+    - name: Change password mechanism2 (root@%)
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "ALTER USER 'root'@'%' IDENTIFIED WITH mysql_native_password BY 'root';"
+    - name: FLUSH PRIVILEGES
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "FLUSH PRIVILEGES;"
+    - name: Show password plugins2
+      run: mysql -h 127.0.0.1 --port 3306 -uroot -proot -e "SELECT user, host, plugin FROM mysql.user;"
+    #
+    # End caching_sha2_password workaround.
+    #
```

Comment:
What does this mean?

Reply:
It meant the mark of the end of the workaround.

https://github.com/embulk/embulk-output-jdbc/blob/8a99944fb54fe7edee7f7de47c8a1d4807da9a45/.github/workflows/check.yml#L81-L96

Reply:
Rather than such a beginning mark and an end mark, I'd suggest to leave individual comments for each step.

Reply:
Removed beginning mark and an end mark